CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## Unreleased
 <!-- Add new, unreleased items here. -->
 
+## 6.4.2 - 2017-11-27
```

Comment:
don't forget to include: https://github.com/Polymer/web-component-tester/commit/3d11569fcb09d422279a789254ee297a397d65ac :)

Reply:
Updated